src/main/java/de/tum/in/www1/artemis/web/rest/ResultResource.java

```diff
@@ -276,4 +280,39 @@ public ResponseEntity<Result> createResultForExternalSubmission(@PathVariable Lo
         return ResponseEntity.created(new URI("/api/results/" + savedResult.getId()))
                 .headers(HeaderUtil.createEntityCreationAlert(applicationName, true, ENTITY_NAME, savedResult.getId().toString())).body(savedResult);
     }
+
+    /**
+     * GET exercises/:exerciseId/feedback-details : Retrieves all negative feedback details and the latest result IDs for a given exercise.
+     *
+     * @param exerciseId The ID of the exercise for which feedback details and result IDs should be retrieved.
+     * @return A ResponseEntity containing a list of all feedback details and the corresponding result IDs for the exercise.
+     */
+    @GetMapping("exercises/{exerciseId}/feedback-details")
+    @EnforceAtLeastEditorInExercise
+    public ResponseEntity<FeedbackDetailsWithResultIdsDTO> getAllFeedbackDetailsForExercise(@PathVariable Long exerciseId) {
+        log.debug("REST request to get all Feedback details for Exercise {}", exerciseId);
+
+        List<StudentParticipation> participations = studentParticipationRepository
+                .findByExerciseIdWithLatestAutomaticResultAndFeedbacksAndTestCasesWithoutIndividualDueDate(exerciseId);
+        removeSubmissionAndExerciseData(participations);
```

Comment:
This endpoint returns a DTO, so this call seems unnecessary.

Reply:
but i need the participations somehow. how should i else retreive them?

Reply:
My next PR was already planned to refactor the table a little bit